chore(docs): add custom seed paths documentation (#29512)
supabase/supabase: apps/docs/content/guides/cli/local-development.mdx

Now that you are managing your database with migrations scripts, it would be great have some seed data to use every time you reset the database.

> For this by default, you can use the seed script in `supabase/seed.sql`.

Reviewer: ```suggestion
For this, you can create a seed script in `supabase/seed.sql`.
```
